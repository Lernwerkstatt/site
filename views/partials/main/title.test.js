const title = require("./title.hbs");

describe("title.hbs", () => {
  it("should match the snapshot", () => {
    expect(title()).toMatchSnapshot();
  });
});