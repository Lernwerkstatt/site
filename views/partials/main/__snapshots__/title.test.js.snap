// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title.hbs should match the snapshot 1`] = `
"<div class=\\"jumbotron\\">
  <div class=\\"row\\">
    <div class=\\"col-12 col-sm-7\\">
      <h1 class=\\"h4\\"></h1>
      <p></p>
    </div>
    <div class=\\"col-12 col-sm-5 align-self-center\\">
      <img src=\\"img/logo.png\\" class=\\"img rounded-circle float-right\\" alt=\\"Die Lernwerkstatt Logo\\" style=\\"max-width:20%; height:auto\\">
    </div>
  </div>
</div>
"
`;
